### ISSUE TITLE
Ensure correct scheduling when exporting

### ISSUE THREAD

**Issue by aleksati** [balintlaczko/VideoAnalysis#61]
The only issue I´ve encountered with this is getting the correct croppings when batch-exporting. Therefore, I think the issue is in the relationship between the [umenu] and the [jit.qt.movie]. What is called first, and so on. I will test to see if this corrects the error and ensures propper cropping when batch-exporting.

**Comment by AleksanderTidemann** [comment]
This is especially relevant when we want to save video settings and recall them when batch-exporting

**Comment by aleksati** [comment]
After some probing, I discovered that a few [deferlow]´s in key places and some "streamlining" from the [umenu] to the [jit.qt] etc., seemed to do the trick. At least on Mac. Tested it with files and batch-exporting with 6 videos. Worked like a charm every time. Even the artifacts cause by [jit.mean] seem to have disappeared. We should probe and test more tomorrow and make it even better, but this seems very promising.

Changes are made to the main file (VideoAnalysis_with_abs) and some abstractions. The changes/added objects are coloured in red, @balintlaczko .
